rskj-core/src/main/java/co/rsk/peg/union/UnionResponseCode.java

```diff
@@ -0,0 +1,26 @@
+package co.rsk.peg.union;
+
+public enum UnionResponseCode {
+    SUCCESS(0),
+    UNAUTHORIZED_CALLER(-1),
+    // Invalid value when The amount requested, combined with previously requested amounts, exceeds the current locking cap value.
+    // or invalid value when increasing union bridge locking cap
+    INVALID_VALUE(-2),
+    // Response codes when enabling or disabling union bridge operations
```

Comment:
```suggestion
    // Enabling already enabled or disabling already disabled union bridge operations
```
something like that?

Reply:
Updated